kernel/src/hil/sensors.rs

```diff
@@ -233,3 +233,41 @@ pub trait PressureClient {
     /// Returns the value in hPa.
     fn callback(&self, pressure: Result<u32, ErrorCode>);
 }
+
+/// Client for receiving NMEA sentences
+pub trait NmeaClient {
+    /// Called when a full NMEA sentence has been found, or if an error
+    /// occurs.
+    ///
+    /// The possible errors are:
+    /// - `BUSY`: Indicates that the hardware is busy with an existing
+    ///           operation or initialisation/calibration.
+    /// - `NOACK`: Failed to correctly communicate over communication protocol.
+    /// - `NOSUPPORT`: Indicates that this data type isn't supported.
+    fn callback(&self, buffer: &'static mut [u8], len: usize, status: Result<(), ErrorCode>);
+}
+
+/// A basic interface for a Nmea sentence reader
+pub trait NmeaDriver<'a> {
```

Comment:
What does driver mean here?

Reply:
The same as all the other uses in the file. It's the trait to read NMEA sentences